### ISSUE TITLE
Javadocs

### ISSUE THREAD

**Issue by CheezBallzPi** [RoboEagles4828/2018Robot#24]
It's a pain to write and serve little purpose. Read the code, god dang it.

Discussion below _that's probably gonna be super short or something but who cares_.

Also, off topic, but TIL you can add checkboxes in Markdown.

- [x] Ooh
- [x] Checkboxes
- [x] Fun times.

**Comment by jacksondm33** [comment]
DriveTrain javadocs are done. You might want to review/edit them if you really care.

**Comment by CheezBallzPi** [comment]
Yeah, made this issue because I saw you were writing javadocs. If we don't need them, we might as well just ditch them entirely.

**Comment by CheezBallzPi** [comment]
and not waste more time writing them, unless you like writing javadocs?

**Comment by jacksondm33** [comment]
I don't like writing javadocs, but there aren't too many to do. I know there are a lot of classes but drivetrain was the biggest and most confusing (to someone who doesn't know our code) one. I wouldn't remove the javadocs we have but only do new ones for entire classes.
